docs/contributing/coding-standards.md

```diff
@@ -94,5 +94,10 @@ Besides the rules that are checked by automatic tools, we have few rules for whi
     - Controllers
     - Facades
     - Repositories
+    - whole namespaces
+        - `Shopsys\FrameworkBundle\Model`
+        - `Shopsys\FrameworkBundle\Component`
+        - `Shopsys\FrameworkBundle\Controller`
+        - `Shopsys\FrameworkBundle\DataFixtures\Demo`
```

Comment:
This whole section is under the heading *Standards that are not checked automatically*. As it is checked automatically, I would rather remove this bullet altogether (starting by `- Visibility of all properties, methods and constants of`).

Reply:
Right, I didn't notice that. As I said in https://github.com/shopsys/shopsys/pull/788#discussion_r254157257, I deleted it and I will discuss this topic with product owner.